rlclientlib/ranking_event.h

```diff
@@ -123,11 +124,11 @@ namespace reinforcement_learning {
     const std::string& get_event_id() const;
 
   public:
-    static multi_slot_decision_event request_decision(const std::string& event_id, const char* context,
+    static multi_slot_decision_event request_decision(const std::string& event_id, string_view context,
```

Comment:
Should event_id be a string_view?

Reply:
see comment in description :)

Reply:
Ah I see. I vote for event_id being done as well, otherwise it feels inconsistent.

Reply:
yeah I agree but it's too much for one PR I think